frontend/templates/product/sections/FeaturedProductsSection/index.tsx

```diff
@@ -23,9 +23,15 @@ import {
    PageContentWrapper,
    ProductVariantPrice,
 } from '@ifixit/ui';
-import { Product } from '@models/product';
+import {
+   fetchProductReviews,
+   Product,
+   ProductReviewData,
+} from '@models/product';
 import { ImagePlaceholder } from '@templates/product/components/ImagePlaceholder';
 import NextLink from 'next/link';
+import { useAppContext } from '@ifixit/app';
+import React, { useState, useEffect } from 'react';
```

Comment:
I think in this repo we've been following the pattern of
```suggestion
import * as React from 'react';
```

And then using the hooks like `React.useState` and `React.useEffect`. It probably doesn't matter, but just thought I'd share the pattern we've been using.